### DISCUSSION TITLE
Reloading The Integration

### DISCUSSION THREAD

**Discussion by AlexKusnezov** [frenck/spook#632] [Q&A]
Hey guys when I reload the integration all of the sensors become unavailable and never come back until I restart HA. Is this a bug or intended?

**Comment by Choleriker** [comment]
Yes, I'm searching for something similar. I want to restart failed integrations. That happens when once a day my router restarts, then some integrations are in state failed. That I want to fix with a script where I can find failed integrations and restart them.

**Comment by RezzZ** [comment]
in case someone stubles uppon this thread. We can reload HA config entries:
<img width="927" alt="Monosnap Home Assistant - Settings – Home Assistant 2025-06-17 10-48-54" src="https://github.com/user-attachments/assets/b240a7e2-57b7-4e93-bb6f-f33b7a86e8d6" />
